src/global-shortcuts.c

```diff
@@ -566,18 +566,85 @@ handle_list_shortcuts (XdpDbusGlobalShortcuts *object,
   return G_DBUS_METHOD_INVOCATION_HANDLED;
 }
 
+static void
+shortcuts_configure_cb (GObject *source_object,
+                        GAsyncResult *res,
+                        gpointer data)
+{
+  g_autoptr(GDBusMethodInvocation) invocation = data;
+  g_autoptr(GError) error = NULL;
+
+  if (!xdp_dbus_impl_global_shortcuts_call_configure_shortcuts_finish (impl, res, &error))
+    {
+      g_dbus_error_strip_remote_error (error);
+      g_warning ("Failed to configure shortcuts: %s", error->message);
+      g_dbus_method_invocation_return_gerror (invocation, error);
+      return;
+    }
+
+  xdp_dbus_global_shortcuts_complete_configure_shortcuts (XDP_DBUS_GLOBAL_SHORTCUTS (global_shortcuts), invocation);
+}
+
+static XdpOptionKey global_shortcuts_configure_shortcuts_options[] = {
+  { "activation_token", G_VARIANT_TYPE_STRING, NULL },
+};
+
+static gboolean
+handle_configure_shortcuts (XdpDbusGlobalShortcuts *object,
+                            GDBusMethodInvocation *invocation,
+                            const gchar *arg_session_handle,
+                            const gchar *arg_parent_window,
+                            GVariant *arg_options)
```

Comment:
`gchar` → `char`. Also the code style we've been using recently is:

```suggestion
static gboolean
handle_configure_shortcuts (XdpDbusGlobalShortcuts *object,
                            GDBusMethodInvocation  *invocation,
                            const char             *arg_session_handle,
                            const char             *arg_parent_window,
                            GVariant               *arg_options)
```

This applies to the other function added in this patch as well.